From dae56fd5c222a0740894b458fbcd25955b410e13 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 3 Jul 2017 14:21:59 +0800
Subject: [PATCH 2069/5242] MLK-15322-7 ARM64: dts: imx8mq-evk: enable mipi
 dsi on evk board

commit  0aff2a3076b9651d186fc54bebc291a52a069d0c from
https://source.codeaurora.org/external/imx/linux-imx.git

Enable mipi dsi on imx8mq evk board by default.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 22e27c7..4525146 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -335,6 +335,12 @@
 	};
 };
 
+&mipi_dsi {
+	reset = <&src>;
+	mux-sel = <&gpr>;	/* lcdif or dcss */
+	status = "okay";
+};
+
 &uart3 { /* BT */
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_uart3>;
-- 
1.7.9.5

